github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2306,7 +2306,12 @@ presubmits:
         - automation/test.sh
         env:
         - name: TARGET
-          value: k8s-1.26-sig-operator-upgrade
+          value: k8s-1.26-centos9-sig-operator-upgrade
+        - name: KUBEVIRT_PSA
+          value: "true"
+        # FIXME: remove after 0.59 release
+        - name: PREVIOUS_RELEASE_TAG
+          value: v0.59.0-rc.0
```

Comment:
@dhiller @brianmcarey We need to keep an eye on this.

Reply:
Created https://github.com/kubevirt/project-infra/issues/2618 to track it